instrumentation/net/http/otelhttp/client.go

```diff
@@ -0,0 +1,58 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelhttp
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+var DefaultClient = &http.Client{Transport: NewTransport(http.DefaultTransport)}
```

Comment:
```suggestion
// DefaultClient is the default Client and is used by Get, Head, Post and PostForm.
// Some changes in OTel might not affect the DefaultClient, such as setting a new global propagator.
// Setting up a new client to apply changes.
var DefaultClient = &http.Client{Transport: NewTransport(http.DefaultTransport)}
```

Don't forget the comment. ;)